clr-k8s-examples/setup_system.sh

```diff
@@ -7,7 +7,6 @@ ADD_NO_PROXY="10.244.0.0/16,10.96.0.0/12"
 ADD_NO_PROXY+=",$(hostname -I | sed 's/[[:space:]]/,/g')"
 
 #Install kubernetes and crio
-sudo -E swupd repair --picky -m 30270 --force
```

Comment:
Thanks. We should find a way for the user to specify this. Will add that to the vagrant setup and passs it down to the script from there.

Reply:
@mcastelino you ok dropping this change? Not sure why we had it hard-coded here, but we should probably come up with a more elegant solution if this is something we will need.

Reply:
Yes. This was done when kubernetes was broken on clear. The better way to do this is to ensure that the scripts included in the clearlinux cloud-native-basic bundle work with released clearlinux. The clearlinux CI is now validating the the scripts. So this change can now be removed.

Reply:
@mcastelino Do we want to add back swupd update that was there in place of this?